lldb/test/API/tools/lldb-dap/stop-events/TestDAP_stopEvents.py

```diff
@@ -0,0 +1,91 @@
+"""
+Test lldb-dap setBreakpoints request
+"""
+
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+import lldbdap_testcase
+from lldbsuite.test import lldbutil
+
+
+class TestDAP_stopEvents(lldbdap_testcase.DAPTestCaseBase):
+    @skipIfWindows
+    @skipIfRemote
```

Comment:
You don't need `@skipIfRemote`. That decorator is automatically applied to all DAP functions.